soc/arm/nordic_nrf/nrf51/power.c

```diff
@@ -1,11 +1,96 @@
 /*
- * Copyright (c) 2018 Intel Corporation.
+ * Copyright (c) 2017 Intel Corporation.
```

Comment:
shouldn't you add nordic copyrights after such significant change?

Reply:
This file is almost exact copy of the soc/arm/nordic_nrf/nrf52/power.c. I kept the original copyright of the source.